src/client.ts

```diff
@@ -75,7 +75,10 @@ export class Client {
       ...clientConfiguration,
       secret: this.#getSecret(clientConfiguration),
     };
-    this.#url = `${this.clientConfiguration.endpoint.toString()}query/1`;
+    this.#url = new URL(
```

Comment:
will this work in node without an import?

Reply:
Yes! `URL` is built in to browsers and node etc.

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/URL/URL#browser_compatibility

Reply:
looks like its global: https://nodejs.org/api/globals.html#url so yes.

Reply:
Oh but valid question if it's available in the global scope. The answer to that is also yes.

Reply:
haha beat me to it 👍🏻